### DISCUSSION TITLE
what is the difference between vmthook and vmtswap?

### DISCUSSION THREAD

**Discussion by Metaphysical1** [danielkrupinski/Osiris#3225] [Q&A]
Osiris on linux defaults to use vmtswap, how is it compared against vmthook?

**Comment by aoqia194** [comment]
RE-EDIT

The information I provided on this was inaccurate.

**Comment by Sumandora** [comment]
I looked at the code. From what I can tell one of them just changes the entry in the vmt and the other one copies the entire vmt and changes stuff. This shouldn't make a difference in functionality and detection. Only thing I could think of is that vmtswap would be unnoticeable faster. Oh and vmthook seems to be incompatible with Linux when vmtswap isn't (at least there is no mention of it in the code)
